torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1280,6 +1280,14 @@ RegisterOperators reg2({
       CREATE_LIST_OPS("Tensor", TensorList),
       CREATE_LIST_OPS("t", GenericList),
   #undef CREATE_LIST_OPS
+      Operator("aten::len(...) -> int", [](const Node* node) {
```

Comment:
What is the ... syntax exactly ? Just for tuples ?

Reply:
It's for varargs so that this works for all tuples